Audacia.Quest/Audacia.Quest/wwwroot/css/app.css

```diff
@@ -1,64 +1,8 @@
-﻿@import url('open-iconic/font/css/open-iconic-bootstrap.min.css');
-
-html, body {
-    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
-}
-
-h1:focus {
-    outline: none;
-}
-
-a, .btn-link {
-    color: #0071c1;
+﻿body {
+    margin: 0;
+    padding: 0;
 }
 
-.btn-primary {
-    color: #fff;
-    background-color: #1b6ec2;
-    border-color: #1861ac;
+.hide {
+    display: none !important;
```

Comment:
Might be okay, but I would recommended to look into this I personally think that adding `!important` to css attributes can cause issues